Driverless AI/get-started-with-open-source-custom-recipes-tutorial/get-started-with-open-source-custom-recipes-tutorial.md

```diff
@@ -238,15 +238,15 @@ The goal of importing the new transformer recipe is to see if we can further opt
 
 6\. Copy and paste  the following URL for the SumTransformer into the **Load Custom Recipe** box then click on **Save.**  
 
-**Note:** The Driverless AI Recipes branches rel-1.8.7 and rel-1.8.7.1 are the same, and have been merged into rel-1.8.7; that is why we are using the links for the 1.8.7 branch.
+**Note:** The Driverless AI Recipe branches rel-1.9.0 and rel-1.9.0 are the same, and have been merged into rel-1.9.0; that is why we are using the links for the 1.9.0 branch.
```

Comment:
Please delete this note. This note was applicable to the 1.8.7.1  release because there were two branches merged for the recipes (1.8.7 and 1.8.7.1). 1.9.0 currently only has one branch.